.gitignore

```diff
@@ -3,6 +3,6 @@ npm-debug.*
 /config*
 !/config.dist
 /test/coverage
-
 /log*
 /dev
+/.idea
```

Comment:
Add this to your glone git ignore, this file should only contain ignores related to the project itself.